agent/engine/docker_task_engine_unix_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows,!integration
+// +build linux,!integration
```

Comment:
I think there is still value in running these tests on OSX. It might be worth only importing the linux-specific code in a different test file, like `docker_task_engine_linux_test.go`